<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home_window</class>
 <widget class="QDialog" name="home_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListView" name="all_images">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>280</y>
     <width>251</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_holder">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>560</width>
     <height>560</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>ncoins.jpg</pixmap>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="getImages">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>250</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Images</string>
   </property>
  </widget>
  <widget class="QPushButton" name="request_image">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>590</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Request Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_image">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>620</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>View Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>590</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>No Image Requested</string>
   </property>
  </widget>
  <widget class="QLabel" name="views_num">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>620</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>No. of Views 0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
